bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/DbLedgerStorageStats.java

```diff
@@ -60,6 +60,7 @@
     private static final String FLUSH_LEDGER_INDEX = "flush-ledger-index";
     private static final String FLUSH_SIZE = "flush-size";
     private static final String THROTTLED_WRITE_REQUESTS = "throttled-write-requests";
+    private static final String THROTTLED_WRITE = "throttled-write";
```

Comment:
we'd better use `throttled-write-time` instead of `throttled-write` as metric name ?

Reply:
throttled-write is OpStatsLogger, this prometheus metrics has two method: sum and count
1.  throttled_write:  throttled's latency
2.  throttled_write_count: the count of throttled
3.  throttled_write_sum: the sum throttled's latency